src/dos/drive_overlay.cpp

```diff
@@ -832,15 +835,15 @@ bool Overlay_Drive::FindNext(DOS_DTA & dta) {
 
 
 bool Overlay_Drive::FileUnlink(char * name) {
-//TODO check the basedir for file existence in order if we need to add the file to deleted file list.
-	Bit32u a = GetTicks();
-	if (logoverlay) LOG_MSG("calling unlink on %s",name);
+	// TODO check the basedir for file existence in order if we need to add the file to deleted file list.
+	auto a = GetTicks();
```

Comment:
Looks like all of this `a` stuff follows the same pattern (I won't comment more on it; might as well get it all)